### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by aaronovz1** [Tencent/rapidjson#1546]
Can we get a new release? The last one is 3 years old.

**Comment by gdsotirov** [comment]
I would like to ask the same. The recently released MySQL 8.0.17 requires RAPIDJSON_WITH_STD_REGEX, which is not available in version 1.1.0 and this would make dependency management hard.

**Comment by voyaxhzhang** [comment]
hello. I saw the code is consistent in the update but there is no release version.  we have a trouble when  Introducing the library.
Can we get a new version?

**Comment by SaberUK** [comment]
A release would be greatly appreciated. We are interested in using RapidJSON but the lack of releases gives us doubt as to whether it is maintained enough for us to risk depending on in a project with long release cycles.

**Comment by c0nd3v** [comment]
Yes, this is very worrying and am thinking of using another library instead. Could we at least get an official statement on this?

**Comment by aaronovz1** [comment]
Does anyone have insight into why a release hasn't been cut for so many years?

**Comment by eiband** [comment]
I am also very interested in a new release.

**Comment by c0nd3v** [comment]
@miloyip

**Comment by wwizz** [comment]
@miloyip perhaps you can share your thoughts on creating a new release tag? Is there a specific reason for not releasing. 

(Any tag would be fine, even if it included something like 'beta' eg 1.1.1-beta1 it wil help integration a lot)

Thanks

**Comment by breznak** [comment]
@tencent-adm hi, could you please update us in this issue of a new release? Is this project mantained and can we expect any release(s) soon? Thank you

**Comment by delmyers** [comment]
@miloyip you have managed previous rapidjson releases. Is there going to be a new one?

**Comment by aut0** [comment]
FWIW, this is the reason why I decided not to use this library and went for [this](https://github.com/nlohmann/json) instead.

**Comment by Livius90** [comment]
Use nlohmann C++ json lib, it is much better in releasing and documentation. I gave up waiting for new rapidjson release, i am using this other from now.
https://github.com/nlohmann/json
